### DISCUSSION TITLE
Sony Yay Hindi

### DISCUSSION THREAD

**Discussion by Achu2234** [iptv-org/community#806] [Q&A]
### Channel Name

Sony Yay hindi

### Channel ID

SonyYay.in

### Website

https://www.sonyyay.com/in-en/home

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19785
